src/pcm-sensor-server.cpp

```diff
@@ -427,6 +427,10 @@ class JSONPrinter : Visitor
         PCM* pcm = PCM::getInstance();
         printCounter( "DRAM Writes",                   getBytesWrittenToMC    ( before, after ) );
         printCounter( "DRAM Reads",                    getBytesReadFromMC     ( before, after ) );
+
+        if (pcm->localMemoryRequestRatioMetricAvailable())
+            printCounter( "DRAM Local Percentage",         getLocalMemoryRequestRatio( before, after ) );
+
```

Comment:
I think the addition of a new metric should be split out into a separate merge request or at least a separate commit.